tests/test_e2e_ocr_conversion.py

```diff
@@ -59,6 +61,10 @@ def test_e2e_conversions():
         TesseractCliOcrOptions(force_full_page_ocr=True),
     ]
 
+    # only works on mac
+    if "darwin" == sys.platform:
+        engines.append(OcrMacOptions(force_full_page_ocr=True))
```

Comment:
Let's add both `OcrMacOptions()` and `OcrMacOptions(force_full_page_ocr=True)`

Reply:
I've updated the code to include both `OcrMacOptions()` and `OcrMacOptions(force_full_page_ocr=True)` as requested.